Xbim.Tessellator/XbimTessellator.cs

```diff
@@ -161,6 +159,9 @@ private XbimShapeGeometry MeshPolyhedronBinary(IIfcTriangulatedFaceSet triangula
                         var rect = new XbimRect3D(pt.A, pt.B, pt.C, 0, 0, 0);
                         bb.Union(rect);
                     }
+
+                    binaryWriter.Write(facesCount);

```

Comment:
The actual change. Shifted from line 151 to 163 due to a wrong order within the storage sequence.

Reply:
Thanks for spotting this out. I think we hardly ever got his in the real world models.